src/content/en/2024/third-parties.md

```diff
@@ -3,17 +3,205 @@
 title: Third Parties
 description: Third Parties chapter of the 2024 Web Almanac covering data of what third parties are used on the web and a deep dive into preventing them causing negative performance impacts.
 hero_alt: Hero image of Web Almanac characters plugging various things into a web page.
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [turban1988, Yash-Vekaria, zubairshafiq, ChrisBeeti]
+reviewers: [tunetheweb]
+editors: [neriiavr]
+analysts: [Yash-Vekaria, ChrisBeeti]
 translators: []
+turban1988_bio: TODO
+Yash-Vekaria_bio: TODO
+zubairshafiq_bio: TODO
+ChrisBeeti_bio: TODO
```

Comment:
@ChrisBeeti can you please provide a short bio about yourself?

Reply:
```suggestion
ChrisBeeti_bio: Chris Böttger is a PhD candidate in Computer Science at the [Westphalian University of Applied Science](https://www.en.w-hs.de/). His research focuses on web and network security, primarily focusing on user privacy and tracking technologies.
```